tests/letstest/apache2_targets.yaml

```diff
@@ -1,3 +1,5 @@
+# These images are located in us-east-1 when possible.
```

Comment:
```suggestion
# These images are located in us-east-1.
```
My understanding is these images must be located in us-east-1 because we configure the tests to run in that region and AMIs are specific to the region. Documentation on this is somewhat hard to find, but https://aws.amazon.com/premiumsupport/knowledge-center/copy-ami-region/ talking about how to copy an AMI from one region to another and all AMI listings I've ever seen having different AMI IDs for each region seem to suggest this to me.

If you agree, I think we should make this change on the other file too.

Reply:
Oh my god, I always assumed that, like DO, this is just for speed-up purposes. Your explanation matches the docs though, I'll fix it.